modules/apps/transfer/keeper/relay_test.go

```diff
@@ -321,7 +323,7 @@ func (suite *KeeperTestSuite) TestOnRecvPacket() {
 		{
 			"failure: receive on module account",
 			func() {
-				receiver = suite.chainA.GetSimApp().AccountKeeper.GetModuleAddress(types.ModuleName).String()
+				receiver = suite.chainB.GetSimApp().AccountKeeper.GetModuleAddress(types.ModuleName).String()
```

Comment:
This was also wrong, but it worked because the module address of transfer for chain A and chain B is the same.